ci/tests.py

```
@@ -30,13 +31,17 @@
 
 
 def _availability_zones():
```

this method should be used only when having to deal with openstack backend

---

but for now, we can suppress the exception and return an empty array